### PR TITLE
Extend documentation for security

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: docs/source/security.rst [hunk 1]
```diff
@@ -4,35 +4,76 @@ Thoth Security Advises
...
 as a normalizer for security score. Having a single high-severity,
 high-confidence issue in a small Python project is much more concerning than if
 a single high-severity, high-confidence issue is found in a large project.
 
+Using security advises in OpenShift S2I
+#######################################
+
+Thoth's integration in OpenShift S2I can block building Python applications
+that are potentially vulnerable. By adjusting ``recommendation_type`` to
+``security``, the build process fails if a any package is considered
```
Comment (KPostOffice) [inline docs/source/security.rst:68]:
```suggestion
``security``, the build process fails if any package is considered
```

**Review by KPostOffice — COMMENTED** [review]
One comment

**Comment by fridex** [comment]
> One comment

Adjusted based on the review.

/approve

**Pull request merged**